fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -904,6 +906,11 @@ public static PartitionData partitionDataFromPath(String relativePartitionPath,
                     field.name().equals(parts[0]), "Invalid partition: %s", partitions[i]);
 
             org.apache.iceberg.types.Type sourceType = spec.partitionType().fields().get(i).type();
+            // apply url decoding for string/fixed type
+            if (sourceType.typeId() == Type.TypeID.STRING || sourceType.typeId() == Type.TypeID.FIXED) {
```

Comment:
what wiil happen if partition value has special characters? such like '09%01%2020'

Reply:
Can trino and spark read data written by sr?

Reply:
Yes.